lib/bitcoin/payment_code.rb

```diff
@@ -0,0 +1,82 @@
+module Bitcoin
+
+  # BIP47 payment code
+  class PaymentCode < ExtKey
+    attr_accessor :x_value
+
+    VERSION_BYTE = '47'
+    SUPPORT_VERSIONS = ['01']
+    SUPPORT_SIGNS = ['02', '03']
+
+    def initialize
+      @version = '01'
+      @features_bits = '00'
+      @sign = '02'
```

Comment:
In this implementation, y coordinate of the compressed public key is always positive.